lib/permutations.rb

```diff
@@ -1,4 +1,16 @@
 
 def permutations?(string1, string2)
-  
+  if string1.length != string2.length
+    return false
+  end
+  hash_holder = {}
+  string1.chars.each do |char|
+    hash_holder[char] = true
+  end
+  string2.chars.each do |char|
+    if !hash_holder[char]
+      return false
+    end
```

Comment:
This checks to see if the strings are the same, but not necessarily in the same quantity. For C13, I've updated my tests to check for this.